.github/workflows/Makefile

```diff
@@ -7,20 +7,41 @@ CODE_TRANSFORMS=code2parquet code_quality header_cleanser malware proglang_selec
 LANG_TRANSFORMS=doc_chunk doc_quality lang_id pdf2parquet pii_redactor text_encoder
 
 
+# A list that holds transforms that should not be tested with KFP
+KFP_BLACK_LIST="doc_chunk,pdf2parquet,pii_redactor"
+
 transform-tests:
-	$(MAKE) TRANSFORM_SUBDIR=universal .transform-tests
-	$(MAKE) TRANSFORM_SUBDIR=language .transform-tests
-	$(MAKE) TRANSFORM_SUBDIR=code .transform-tests
+	$(MAKE) TRANSFORM_SUBDIR=universal .transform-tests .transform-kfp-tests
```

Comment:
should it be `$(MAKE) TRANSFORM_SUBDIR=universal .transform-tests ` ?

Reply:
Fixed. Thanks